### PR TITLE
Nette 3.0

### PR DESCRIPTION
**Pull request by miranovy**
Updated to Nette 3.0

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,16 +12,16 @@
   ],
   "require": {
     "php": ">= 7.1",
-    "nette/di": "~2.4.10",
-    "nette/utils": "~2.5.2",
-    "latte/latte": "~2.4.7|~2.5.1",
+    "nette/di": "^3.0",
```
Comment (f3l1x) [inline composer.json]: Please ^3.0.0 and others same way.

File: src/DI/ParsedownExtraExtension.php [hunk 2]
```diff
@@ -6,47 +6,50 @@
...
  * ParsedownExtra Extension
  */
 class ParsedownExtraExtension extends CompilerExtension
 {
 
-	/** @var mixed[] */
-	private $defaults = [
-		'helper' => 'parsedown',
-	];
+	public static function createSchema(): Schema
```
Comment (f3l1x) [inline src/DI/ParsedownExtraExtension.php]: Why do you need static fnc?

**Review by f3l1x — CHANGES_REQUESTED** [review]
Thanks for an PR. I have some points.

File: composer.json (hunk 1 above)
Comment (mabar) [reply composer.json]: ~3.0.0, to be exact

File: src/DI/ParsedownExtraExtension.php (hunk 2 above)
Comment (mabar) [reply src/DI/ParsedownExtraExtension.php]: It's copied from a package where we compose extensions into single root extension, not needed here

**Comment by f3l1x** [comment]
Done @miranovy https://github.com/contributte/latte-parsedown-extra/releases/tag/1.5.0

**Pull request merged**